docs/guides/creating-interactive-workflows.md

```diff
@@ -162,3 +213,200 @@ def get_shirt_order():
+    async for name_input in receive_input(str, with_metadata=True, timeout=None):
+        await name_input.respond(f"Hello, {name_input.value}!")
+```
+
+Cool! There's one problem left: this flow runs forever! We need a way to signal that it should exit. Let's keep things simple and teach it to look for a special string:
+
+```python
+from prefect import flow
+from prefect.input.run_input import receive_input
+
+
+
+EXIT_SIGNAL = "__EXIT__"
+
+
+@flow
+async def greeter():
+    async for name_input in receive_input(
+        str, with_metadata=True, poll_interval=0.1, timeout=None
+    ):
+        if name_input.value == EXIT_SIGNAL:
+            print("Goodbye!")
+            return
+        await name_input.respond(f"Hello, {name_input.value}!")
+```
+
+With a `greeter` flow in place, now we're ready to create the flow that sends `greeter` names!
+
+### Sending input
+
+You can send input to a flow with the `send_input` function. This works similarly to `receive_input`, taking the same types for the `run_input` argument (basic types like `str`, `BaseModel` subclasses, and `RunInput` subclasses).
+
+!!! note "When can you send input to a flow run?"
+    You can send input to a flow run as soon as you have the flow run's ID. The flow does not have to be receiving input for you to send input. If you send a flow input before it is receiving, it will see your input when it calls `receive_input` (as long as the types in the `send_input` and `receive_input` calls match!)
+
+Next, we'll create a `sender` flow that starts a `greeter` flow run and then enters a loop, continuously getting input from the terminal and sending it to the greeter flow:
+
+```python
+@flow
+async def sender():
+    greeter_flow_run = await run_deployment(
+        "greeter/send-receive", timeout=0, as_subflow=False
+    )
+
+    greetings_seen = set()
+
+    while True:
+        name = input("What is your name? ")
+        if not name:
+            continue
+
+
+        if name == "q" or name == "quit":
+            await send_input(EXIT_SIGNAL, flow_run_id=greeter_flow_run.id)
+            print("Goodbye!")
+            break
+
+        await send_input(name, flow_run_id=greeter_flow_run.id)
+
+        async for greeting in receive_input(
+            str, with_metadata=True, exclude_keys=greetings_seen, timeout=None, poll_interval=0.1
+        ):
+            print(greeting)
+            greetings_seen.add(greeting.metadata.key)
+            break
+```
+
+There's more going on here than in `greeter`, so let's take a closer look at the pieces.
+
+First, we use `run_deployment` to start a `greeter` flow run. This means we must have a worker or `flow.serve()` running in separate process. That process will begin running `greeter` while `sender` continues to execute. Calling `run_deployment(..., timeout=0)` ensures that `sender` won't wait for the `greeter` flow run to complete, because, of course, it's running a loop and will only exit when we send `EXIT_SIGNAL`.
```

Comment:
```suggestion
First, we use `run_deployment` to start a `greeter` flow run. This means we must have a worker or `flow.serve()` running in separate process. That process will begin running `greeter` while `sender` continues to execute. Calling `run_deployment(..., timeout=0)` ensures that `sender` won't wait for the `greeter` flow run to complete, because it's running a loop and will only exit when we send `EXIT_SIGNAL`.
```